mathjax3-ts/output/chtml/CssStyles.ts

```diff
@@ -0,0 +1,113 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2017 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/**
+ * @fileoverview  Implements the CssStyles class for handling stylesheets
+ *
+ * @author dpvc@mathjax.org (Davide Cervone)
+ */
+
+/*
+ * The data for a selector
+ */
+export type StyleData = {
+    [property: string]: string | number;
+};
+
+/*
+ * A list of selectors and their data (basically a stylesheet)
+ */
+export type StyleList = {
```

Comment:
That's not really a list. Maybe StyleSet?

Reply:
It's as much a list as `ProperList`, `OptionList`, and the other `*List` types that we have elsewhere.  Do you really think it needs to change?

Reply:
If it's consistent with the rest, that's fine.